spec/features/user_registration_spec.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.feature 'User Registration', type: :feature do
+  subject { click_button 'Sign up' }
+
+  before do
+    visit new_user_registration_path
+  end
+
+  context 'when user tries submits to the form' do
```

Comment:
Remove this context. It s clear that user tries to submit the form because it is User Registration spec.